lms/static/coffee/fixtures/student_admin.html

```diff
@@ -0,0 +1,148 @@
+
+    <a href="" class="progress-link" data-endpoint="${ section_data['get_student_progress_url_url'] }"> ${_("Student Progress Page")} </a>
+  </p>
+  </div>
+
+  <hr>
+</div>
+
+<div class="student-grade-container action-type-container">
+  <h2>${_("Student-specific grade adjustment")}</h2>
+  <div class="request-response-error"></div>
+  <p>
+  <label>
+  ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+  <input type="text" name="student-select-grade" placeholder="${_("Student Email or Username")}">
+  </label>
+  </p>
+  <br>
+
+  <label> ${_("Specify a problem in the course here with its complete location:")}
+  <input type="text" name="problem-select-single" placeholder="${_("Problem location")}">
+  </label>
+
+  ## Translators: A location (string of text) follows this sentence.
+  <p>${_("You must provide the complete location of the problem. In the Staff Debug viewer, the location looks like this:")}<br/>
+  <code>i4x://edX/Open_DemoX/problem/78c98390884243b89f6023745231c525</code></p>
+
+  <p>
+  ${_("Next, select an action to perform for the given user and problem:")}
+  </p>
+
+  <p>
+  <input type="button" name="reset-attempts-single" value="${_("Reset Student Attempts")}" data-endpoint="${ section_data['reset_student_attempts_url'] }">
+
+
+    <input type="button" name="rescore-problem-single" value="${_("Rescore Student Submission")}" data-endpoint="${ section_data['rescore_problem_url'] }">
+
+  </p>
+
+  <p>
+    <label> ${_('You may also delete the entire state of a student for the specified problem:')}
+    <input type="button" class="molly-guard" name="delete-state-single" value="${_("Delete Student State for Problem")}" data-endpoint="${ section_data['reset_student_attempts_url'] }"></label>
+
+  </p>
+
+
+
+    <p id="task-history-single-help">
+      ${_("Rescoring runs in the background, and status for active tasks will appear in the 'Pending Instructor Tasks' table. "
+      "To see status for all tasks submitted for this problem and student, click on this button:")}
+    </p>
+
+    <p><input type="button" name="task-history-single" value="${_("Show Background Task History for Student")}" data-endpoint="${ section_data['list_instructor_tasks_url'] }" aria-describedby="task-history-single-help"></p>
+    <div class="task-history-single-table"></div>
+
+  <hr>
+</div>
+
+
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
+  <div class="request-response-error"></div>
+  <label>
+  ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+  <input type="text" name="entrance-exam-student-select-grade" placeholder="${_('Student Email or Username')}">
+  </label>
+  <br>
+
+  <p>
+  ${_("Next, select an action to perform for the given user's entrance exam grade:")}
```

Comment:
Change to: Select an action for the student's entrance exam. This action will affect every problem in the student's exam.